README.md

```diff
@@ -593,6 +593,7 @@ Libraries to help manage database schemas and migrations.
 * [Drest](https://github.com/leedavis81/drest) - A library for exposing Doctrine entities as REST resource endpoints.
 * [Restler](https://github.com/Luracast/Restler) - A lightweight framework to expose PHP methods as RESTful web API.
 * [wsdl2phpgenerator](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator) - A tool to generate PHP classes from SOAP WSDL files.
+* [API PLATFORM](https://github.com/dunglas/api-platform) - Expose in minutes an hypermedia REST API that works out of the box that embraces JSON-LD, Hydra and provides a ton of features (also usable as standalone symphony bundle).
```

Comment:
API Platform (case)
https://api-platform.com (a dedicated website is in progress)
Symfony